### DISCUSSION TITLE
Convert NuGet-package cration to new SDK format

### DISCUSSION THREAD

**Discussion by teneko** [manuelroemer/Nullable#18] [Ideas]
Hello. I forked your project and converted the NuGet-part to the new SDK format. I have played out my my MSBuild knowledge and well, nothing more than the cspoj is left over. 😃

Because the NuGet-part and the outer build process with Nuke is now obsolete I did not send you a pull request - too much changes.

Maybe you are interested in looking at the solution (not the file) and adopt it by youself if you want. 🙂

Here the link to the fork (it is commited via a squashed-git-subtree-merge here: [Teronis.Nullable](https://github.com/teroneko/Teronis.DotNet/tree/develop/src/Nullable).

**Comment by manuelroemer** [comment]
_C&Ping my reply from [the issue](https://github.com/manuelroemer/Nullable/issues/17) for completeness:_

I'll for sure have a look at it at some point. The Nuke build project is something that I dislike myself. I just can't really tell when as I don't have a lot of spare time on hand recently. 😄

Thanks a lot for the effort and making the code public!

**Reply by teneko** [reply]
Ahh, since I've just mimicked your .nupkg it is maybe worth to change the extensions back from pp to cs and make this: `\<Compile Condition="<is it my file file from package>">...false\</Combpile>` in targets. The default behaviour of nuget with content(Files) would remain.
